pkg/api/types.go

```diff
@@ -652,3 +652,11 @@ const (
 	TriggerSCM  string = "webhook"
 	TriggerCron string = "timer"
 )
+
+// WorkerPod represents some infomation of cyclone worker pod in kubernetes.
+type WorkerPod struct {
```

Comment:
`WorkerPod` is not just for k8s, but also for Docker. What about `WorkerInstance`?

Reply:
done